.github/workflows/swift.yml

```diff
@@ -62,6 +62,9 @@ jobs:
           if [ -e /home/runner/.wasmer/wasmer.sh ]; then
             source /home/runner/.wasmer/wasmer.sh
           fi
+
+          # Cleaning up the build directory is required, because `swift test` fails with `--static-swift-stdlib` on Linux.
```

Comment:
```suggestion
          # Object files built with `--static-swift-stdlib` are not compatible with object files without
          # `--static-swift-stdlib` because they link different libraries. And `swift test --static-swift-stdlib`
          # doesn't work since XCTest is not built as static library and not installed `under swift_static`.
```